aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -69,20 +69,33 @@ void check_allany_for_meta(
     const Tensor& self,
     int64_t dim,
     bool keepdim) {
-  dim = maybe_wrap_dim(dim, self.dim());
   const auto& result = meta.maybe_get_output();
   auto out_dtype = check_allany_and_get_output_dtype(name, self, result, keepdim);
   auto shape = get_reduction_shape(self, dim, keepdim);
   meta.set_output(shape, self.options().dtype(out_dtype));
   namedinference::propagate_names_for_reduction(result, self, dim, keepdim);
 }
 
-TORCH_META_FUNC2(all, dim)(const Tensor& self, int64_t dim, bool keepdim) {
-  check_allany_for_meta(*this, "all", self, dim, keepdim);
+ TORCH_PRECOMPUTE_META_FUNC2(all, dim)
+ (const Tensor& self,
+  int64_t dim,
+  bool keepdim) {
+   dim = maybe_wrap_dim(dim, self.dim());
+   check_allany_for_meta(*this, "all", self, dim, keepdim);
+   return {
+     .dim_=dim
```

Comment:
Is there anything stopping us from just calling this dim (no trailing underscore?)